### PR TITLE
Contributing md tomassirio

### PR DESCRIPTION
**Pull request by tomassirio**
Added the two links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,12 @@ Playbook Exemplo
         - role: leolleocomp.mongodb
           mongodb_configuration_template_file_path: ./templates/mongod-with-nice-tricks.conf.j2
 
+Contributing
```
Comment (o-leolleo) [inline README.md]: I think here the text could be like "Como contribuir?"

File: README.en.md [hunk 2]
```diff
@@ -46,6 +46,13 @@ Example Playbook
         - role: leolleocomp.mongodb
           mongodb_configuration_template_file_path: ./templates/mongod-with-nice-tricks.conf.j2
 
+
+Contributing
```
Comment (o-leolleo) [inline README.en.md]: I think here the text could be "How to contribute?"

**Review by o-leolleo — CHANGES_REQUESTED** [review]
Hey @tomassirio I've added minor suggestions to make the text a bit more friendly, what do you think?

**Comment by tomassirio** [comment]
Leo, could you open another issue with this requests? I'm afraid I might not be able to correct them until the evening (9 AM here in Buenos Aires)

**Comment by o-leolleo** [comment]
Hey @tomassirio, didn't had time until now to see your comment. If you're in a hurry I can accept it, but I think I can apply the suggestions later, not sure if they would fit in an issue.

**Comment by tomassirio** [comment]
Check them now

**Comment by o-leolleo** [comment]
@tomassirio Everything fine, thank you!

**Pull request merged**